lib/LMSManagers/LMSConfigManager.php

```diff
@@ -134,12 +138,12 @@ public function GetConfigDefaultType($section, $var)
         return $type;
     }
 
-    public function CheckOption($type, $value)
+    public function CheckOption($var, $value, $section, $type)
```

Comment:
Przejrzałem starą definicję metody CheckOption(...) - wydaje się ona jakaś niekompletna, bo bierze pod uwagę tylko nazwę zmiennej - można założyć, że to był błąd projektowy i jednak od razu zmienić interfejs tak, żeby było porządnie - nawet nie sugerując się, że gdzieś już jest używane i założyć, że pierwszy parametr to section.variable, a drugi to jego wartość...